aten/src/ATen/native/native_functions.yaml

```diff
@@ -181,14 +181,21 @@
 
 - func: _assert_tensor_metadata(Tensor a, SymInt[]? size=None, SymInt[]? stride=None, ScalarType? dtype=None) -> ()
 
-- func: sym_constrain_range(Scalar size, int? min=None, int? max=None) -> ()
+- func: sym_constrain_range(Scalar size, *, int? min=None, int? max=None) -> ()
   dispatch:
-    CPU: sym_constrain_range_cpu
-    CUDA: sym_constrain_range_cuda
+    CompositeExplicitAutograd: sym_constrain_range
+
+- func: sym_constrain_range_for_size(Scalar size) -> ()
```

Comment:
This naming is weird because there's no range anymore. Why not just call it the public API name (`constrain_as_size`) (I'm also super unclear about why this operators have `sym_` prefix on their name, it seems unnecessary)